newrelic/hooks/framework_graphenedjango.py

```diff
@@ -0,0 +1,102 @@
+# Copyright 2010 New Relic, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import sys
+from inspect import isawaitable
+
+from newrelic.api.error_trace import ErrorTrace
+from newrelic.api.graphql_trace import GraphQLOperationTrace
+from newrelic.api.transaction import current_transaction
+from newrelic.common.object_names import callable_name
+from newrelic.common.object_wrapper import wrap_function_wrapper
+from newrelic.common.package_version_utils import get_package_version
+from newrelic.common.signature import bind_args
+from newrelic.core.graphql_utils import graphql_statement
+from newrelic.hooks.framework_graphql import GRAPHQL_VERSION, ignore_graphql_duplicate_exception
+from newrelic.hooks.framework_graphql_py3 import nr_coro_graphql_impl_wrapper
+
+GRAPHENE_DJANGO_VERSION = get_package_version("graphene_django")
+graphene_django_version_tuple = tuple(map(int, GRAPHENE_DJANGO_VERSION.split(".")))
+
+
+def wrap_GraphQLView_get_response(wrapped, instance, args, kwargs):
+    transaction = current_transaction()
+
+    if not transaction:
+        return wrapped(*args, **kwargs)
+
+    string_response, status_code = wrapped(*args, **kwargs)
+    response = json.loads(string_response)
+    transaction.process_response(status_code, response)
+
+    return string_response, status_code
```

Comment:
I don't believe this instrumentation is necessary. If this is a view, it should be capturing the response headers and status code in the `django` instrumentation. right?

Tried it locally with this commented out, and it still passed 100% of tests. If this is actually necessary for some reason then we need a test that covers this.

Reply:
You're right.  I even added a validator to some tests after removing this to make sure that this still is populated.  Removed!